### DISCUSSION TITLE
Query regarding the GitHub Octernship program

### DISCUSSION THREAD

**Discussion by phanendharreddy** [community/community#51979] [GitHub Education]
Hai, I'm Phanendhar Reddy, 
I've completed my assignment for the GitHub Octernship program and uploaded it in the main and feedback branches. 
The deadline is passed and now I'm waiting for the results. How will I know? that If I am selected for the program? There is another branch called assignment-update. I've seen that branch as well, there is no update. And also I'm concerned about my submission. As per instructions, the code should upload to the feedback branch. I've done it. But just now explored GitHub Classroom. I've seen the Assignment related submission and that's so confusing. Currently, I'm a bit tense about my submission.
Is there any timeline provided for the program? If so please share it with me.
I'm not aware of whom should I approach whether the organization or GitHub or watchers. I've messaged to watchers as well. But, they're probably busy so not responding. 
Any leads from the community are appreciated. 
Thanking you in advance!!

**Comment by samus-aran** [comment]
Hi @phanendharreddy 

We would recommend students working on Octernship projects to reach out to the project’s maintainers to ask for clarification on the proposed tasks or identify any issues they are experiencing. All project’s may have different preferred methods of communication but here are a few places to check where best to raise inquiry:

- Octernships Project’s README - This may include instructions on how and where to raise an inquiry.
- Octernships Project’s Discussions - Not all projects’ repos have this enabled. We encourage students to first search in case your question has already been answered.
- Octernships Project’s Issues - We encourage students to first search in case your question has already been answered. We also encourage students to follow instructions on the issue template(s) on how to create a suitable issue if provided.

Hope this helps 👍

**Comment by phanendharreddy** [comment]
Firstly, Thank you @samus-aran  for your response.
Actually, I've messaged the watchers but they're not responding. The mode for communication is not mentioned in the readme.md file. And I've messaged in GitHub Classroom as well - no response.
